blazor/dropdown-list/sorting.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: Sorting in Blazor DropDown List Component | Syncfusion
+description: Checkout and learn here all about Sorting in Syncfusion Blazor DropDown List component and much more.
+platform: Blazor
+control: DropDown List
+documentation: ug
+---
+
+# Sorting in Dropdown List
+
+The Sorting enables you to sort data in the `Ascending` or `Descending` order. To enable sorting in the DropDownList, set the [SortOrder](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownBase-1.html#Syncfusion_Blazor_DropDowns_SfDropDownBase_1_SortOrder) property to required value. 
+
+The available type of sort orders are.
+
+SortOrder     | Description
+------------ | -------------
+  None       | The data source is not sorting.

```

Comment:
Link the SortOdert enum values API( None, Ascending, Descending) https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SortOrder.html#Syncfusion_Blazor_DropDowns_SortOrder_Ascending